### DISCUSSION TITLE
Github support not answering to verify request?

### DISCUSSION THREAD

**Discussion by ender-schesi** [community/community#23120] [New to GitHub]
<p>My account [enderschesi] recently got flagged falsely as spam.</p>
<p>I then created a ticket to get my account reverified. On the support page it noted that it might take up to 7 Business days for my ticket to get review. I have now been waiting for almost one and a halft Months for someone to look at my account.</p>
<p>I need to get it reviewed as i cant create organizations while flagged as spam.</p>

**Comment by queenofcorgis** [comment]
<p>Welcome <a class="mention" href="/ender-schesi">@ender-schesi</a>, mind sharing the ticket # with me so I can look into this? I can’t promise anything and I’m unable to escalate, but I can make sure it is in the right place!</p>

**Comment by ender-schesi** [comment]
<p>Sure. The Ticket # is 1603561</p>

**Comment by queenofcorgis** [comment]
<p>Alright it looks like this ticket is with the proper team and assigned to a support member so it is in the right place! Almost two months is a long time to wait so I really appreciate your patience.</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p><a class="mention" href="/ender-schesi">@ender-schesi</a> thanks again for your patience - I received confirmation from the Support team that they’ve reviewed your account. Welcome back :stars:</p>

**Reply by sachinDev07** [reply]
my account   has been flagged please help sir to revive it
